roles/common/files/ssh_config

```diff
@@ -0,0 +1,35 @@
+
+Include /etc/ssh/ssh_config.d/*.conf
```

Comment:
Since we do not have an additional configuration, it is better to remove it so that no one manually adds it at that place.

Reply:
Checked and fixed